linkerd/app/core/src/errors.rs

```diff
@@ -149,7 +151,7 @@ impl<ReqB, RspB: Default + hyper::body::HttpBody>
                 let is_grpc = req
                     .headers()
                     .get(http::header::CONTENT_TYPE)
-                    .and_then(|v| v.to_str().ok().map(|s| s.starts_with("application/grpc")))
+                    .and_then(|v| v.to_str().ok().map(|s| s.starts_with("GRPC_CONTENT_TYPE")))
```

Comment:
mmm probably don't want to quote it?

Reply:
agh, thanks for catching that
```suggestion
                    .and_then(|v| v.to_str().ok().map(|s| s.starts_with(GRPC_CONTENT_TYPE)))
```